__tests__/main.js

```diff
@@ -6,17 +6,16 @@ var options = {
 
```

Comment:
can we name the test folder as `tests` or `test` instead of `__tests__`. Haven't seen this convention used in nodejs projects.  Let me know.

Reply:
Lets discuss this in later iterations. Merging the pull.

Reply:
it comes from jest test suite https://facebook.github.io/jest/docs/en/configuration.html#testmatch-array-string, the first time I saw this naming was when I first used yeoman-node generator and from then I decided to stick with this name, but any name (acceptable from jest) it's allright